<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="8" passed="4" failed="4" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-09-16T11:19:49 CEST" name="PositiveLoginTestSuite" finished-at="2021-09-16T11:20:18 CEST" duration-ms="29098">
    <groups>
    </groups>
    <test started-at="2021-09-16T11:19:49 CEST" name="PositiveLoginOnMainPageTest1-CH" finished-at="2021-09-16T11:20:01 CEST" duration-ms="11649">
      <class name="local.HLVTestApp.loginandsignuptests.PositiveLoginTests">
        <test-method is-config="true" signature="setUp(java.lang.String,org.testng.ITestContext)[pri:0, instance:local.HLVTestApp.loginandsignuptests.PositiveLoginTests@7e057f43]" started-at="2021-09-16T11:19:49 CEST" name="setUp" finished-at="2021-09-16T11:19:51 CEST" duration-ms="2364" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[org.testng.TestRunner@55296b50]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="verifySuccessfulLoginActionOnLoginPage(java.util.Map)[pri:0, instance:local.HLVTestApp.loginandsignuptests.PositiveLoginTests@7e057f43]" started-at="2021-09-16T11:19:51 CEST" name="verifySuccessfulLoginActionOnLoginPage" data-provider="csvReader" finished-at="2021-09-16T11:19:52 CEST" duration-ms="736" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{no=1, description=valid username and password 1, password=testPassword1, username=testUsername1}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifySuccessfulLoginActionOnLoginPage -->
        <test-method is-config="true" signature="tearDownAfterEach()[pri:0, instance:local.HLVTestApp.loginandsignuptests.PositiveLoginTests@7e057f43]" started-at="2021-09-16T11:19:52 CEST" name="tearDownAfterEach" finished-at="2021-09-16T11:19:53 CEST" duration-ms="620" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownAfterEach -->
        <test-method is-config="true" signature="setUp(java.lang.String,org.testng.ITestContext)[pri:0, instance:local.HLVTestApp.loginandsignuptests.PositiveLoginTests@7e057f43]" started-at="2021-09-16T11:19:53 CEST" name="setUp" finished-at="2021-09-16T11:19:54 CEST" duration-ms="1266" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[org.testng.TestRunner@55296b50]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="verifySuccessfulLoginActionOnLoginPage(java.util.Map)[pri:0, instance:local.HLVTestApp.loginandsignuptests.PositiveLoginTests@7e057f43]" started-at="2021-09-16T11:19:54 CEST" name="verifySuccessfulLoginActionOnLoginPage" data-provider="csvReader" finished-at="2021-09-16T11:19:55 CEST" duration-ms="692" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{no=2, description=valid username and password 2, password=testPassword2, username=testUsername2}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifySuccessfulLoginActionOnLoginPage -->
        <test-method is-config="true" signature="tearDownAfterEach()[pri:0, instance:local.HLVTestApp.loginandsignuptests.PositiveLoginTests@7e057f43]" started-at="2021-09-16T11:19:55 CEST" name="tearDownAfterEach" finished-at="2021-09-16T11:19:55 CEST" duration-ms="621" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownAfterEach -->
        <test-method is-config="true" signature="setUp(java.lang.String,org.testng.ITestContext)[pri:0, instance:local.HLVTestApp.loginandsignuptests.PositiveLoginTests@7e057f43]" started-at="2021-09-16T11:19:55 CEST" name="setUp" finished-at="2021-09-16T11:19:57 CEST" duration-ms="1247" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[org.testng.TestRunner@55296b50]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="verifySuccessfulLoginActionOnMainPage(java.util.Map)[pri:0, instance:local.HLVTestApp.loginandsignuptests.PositiveLoginTests@7e057f43]" started-at="2021-09-16T11:19:57 CEST" name="verifySuccessfulLoginActionOnMainPage" data-provider="csvReader" finished-at="2021-09-16T11:19:57 CEST" duration-ms="516" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[{no=1, description=valid username and password 1, password=testPassword1, username=testUsername1}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [http://localhost:8080/login] but found [http://localhost:8080/?username=testUsername1&password=testPassword1]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [http://localhost:8080/login] but found [http://localhost:8080/?username=testUsername1&password=testPassword1]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at local.HLVTestApp.loginandsignuptests.PositiveLoginTests.verifySuccessfulLoginActionOnMainPage(PositiveLoginTests.java:36)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifySuccessfulLoginActionOnMainPage -->
        <test-method is-config="true" signature="tearDownAfterEach()[pri:0, instance:local.HLVTestApp.loginandsignuptests.PositiveLoginTests@7e057f43]" started-at="2021-09-16T11:19:58 CEST" name="tearDownAfterEach" finished-at="2021-09-16T11:19:58 CEST" duration-ms="649" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownAfterEach -->
        <test-method is-config="true" signature="setUp(java.lang.String,org.testng.ITestContext)[pri:0, instance:local.HLVTestApp.loginandsignuptests.PositiveLoginTests@7e057f43]" started-at="2021-09-16T11:19:58 CEST" name="setUp" finished-at="2021-09-16T11:20:00 CEST" duration-ms="1321" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[org.testng.TestRunner@55296b50]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="verifySuccessfulLoginActionOnMainPage(java.util.Map)[pri:0, instance:local.HLVTestApp.loginandsignuptests.PositiveLoginTests@7e057f43]" started-at="2021-09-16T11:20:00 CEST" name="verifySuccessfulLoginActionOnMainPage" data-provider="csvReader" finished-at="2021-09-16T11:20:00 CEST" duration-ms="396" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[{no=2, description=valid username and password 2, password=testPassword2, username=testUsername2}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [http://localhost:8080/login] but found [http://localhost:8080/?username=testUsername2&password=testPassword2]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [http://localhost:8080/login] but found [http://localhost:8080/?username=testUsername2&password=testPassword2]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at local.HLVTestApp.loginandsignuptests.PositiveLoginTests.verifySuccessfulLoginActionOnMainPage(PositiveLoginTests.java:36)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifySuccessfulLoginActionOnMainPage -->
        <test-method is-config="true" signature="tearDownAfterEach()[pri:0, instance:local.HLVTestApp.loginandsignuptests.PositiveLoginTests@7e057f43]" started-at="2021-09-16T11:20:00 CEST" name="tearDownAfterEach" finished-at="2021-09-16T11:20:01 CEST" duration-ms="630" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownAfterEach -->
      </class> <!-- local.HLVTestApp.loginandsignuptests.PositiveLoginTests -->
    </test> <!-- PositiveLoginOnMainPageTest1-CH -->
    <test started-at="2021-09-16T11:20:01 CEST" name="PositiveLoginOnMainPageTest2-FF" finished-at="2021-09-16T11:20:18 CEST" duration-ms="17432">
      <class name="local.HLVTestApp.loginandsignuptests.PositiveLoginTests">
        <test-method is-config="true" signature="setUp(java.lang.String,org.testng.ITestContext)[pri:0, instance:local.HLVTestApp.loginandsignuptests.PositiveLoginTests@14fc5f04]" started-at="2021-09-16T11:20:01 CEST" name="setUp" finished-at="2021-09-16T11:20:04 CEST" duration-ms="3550" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[firefox]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[org.testng.TestRunner@2ee4706d]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="verifySuccessfulLoginActionOnLoginPage(java.util.Map)[pri:0, instance:local.HLVTestApp.loginandsignuptests.PositiveLoginTests@14fc5f04]" started-at="2021-09-16T11:20:04 CEST" name="verifySuccessfulLoginActionOnLoginPage" data-provider="csvReader" finished-at="2021-09-16T11:20:05 CEST" duration-ms="810" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{no=1, description=valid username and password 1, password=testPassword1, username=testUsername1}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifySuccessfulLoginActionOnLoginPage -->
        <test-method is-config="true" signature="tearDownAfterEach()[pri:0, instance:local.HLVTestApp.loginandsignuptests.PositiveLoginTests@14fc5f04]" started-at="2021-09-16T11:20:05 CEST" name="tearDownAfterEach" finished-at="2021-09-16T11:20:06 CEST" duration-ms="677" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownAfterEach -->
        <test-method is-config="true" signature="setUp(java.lang.String,org.testng.ITestContext)[pri:0, instance:local.HLVTestApp.loginandsignuptests.PositiveLoginTests@14fc5f04]" started-at="2021-09-16T11:20:06 CEST" name="setUp" finished-at="2021-09-16T11:20:09 CEST" duration-ms="2916" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[firefox]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[org.testng.TestRunner@2ee4706d]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="verifySuccessfulLoginActionOnLoginPage(java.util.Map)[pri:0, instance:local.HLVTestApp.loginandsignuptests.PositiveLoginTests@14fc5f04]" started-at="2021-09-16T11:20:09 CEST" name="verifySuccessfulLoginActionOnLoginPage" data-provider="csvReader" finished-at="2021-09-16T11:20:09 CEST" duration-ms="732" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{no=2, description=valid username and password 2, password=testPassword2, username=testUsername2}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifySuccessfulLoginActionOnLoginPage -->
        <test-method is-config="true" signature="tearDownAfterEach()[pri:0, instance:local.HLVTestApp.loginandsignuptests.PositiveLoginTests@14fc5f04]" started-at="2021-09-16T11:20:09 CEST" name="tearDownAfterEach" finished-at="2021-09-16T11:20:10 CEST" duration-ms="672" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownAfterEach -->
        <test-method is-config="true" signature="setUp(java.lang.String,org.testng.ITestContext)[pri:0, instance:local.HLVTestApp.loginandsignuptests.PositiveLoginTests@14fc5f04]" started-at="2021-09-16T11:20:10 CEST" name="setUp" finished-at="2021-09-16T11:20:13 CEST" duration-ms="2883" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[firefox]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[org.testng.TestRunner@2ee4706d]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="verifySuccessfulLoginActionOnMainPage(java.util.Map)[pri:0, instance:local.HLVTestApp.loginandsignuptests.PositiveLoginTests@14fc5f04]" started-at="2021-09-16T11:20:13 CEST" name="verifySuccessfulLoginActionOnMainPage" data-provider="csvReader" finished-at="2021-09-16T11:20:13 CEST" duration-ms="364" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[{no=1, description=valid username and password 1, password=testPassword1, username=testUsername1}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [http://localhost:8080/login] but found [http://localhost:8080/?username=testUsername1&password=testPassword1]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [http://localhost:8080/login] but found [http://localhost:8080/?username=testUsername1&password=testPassword1]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at local.HLVTestApp.loginandsignuptests.PositiveLoginTests.verifySuccessfulLoginActionOnMainPage(PositiveLoginTests.java:36)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifySuccessfulLoginActionOnMainPage -->
        <test-method is-config="true" signature="tearDownAfterEach()[pri:0, instance:local.HLVTestApp.loginandsignuptests.PositiveLoginTests@14fc5f04]" started-at="2021-09-16T11:20:13 CEST" name="tearDownAfterEach" finished-at="2021-09-16T11:20:14 CEST" duration-ms="623" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownAfterEach -->
        <test-method is-config="true" signature="setUp(java.lang.String,org.testng.ITestContext)[pri:0, instance:local.HLVTestApp.loginandsignuptests.PositiveLoginTests@14fc5f04]" started-at="2021-09-16T11:20:14 CEST" name="setUp" finished-at="2021-09-16T11:20:17 CEST" duration-ms="2869" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[firefox]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[org.testng.TestRunner@2ee4706d]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="verifySuccessfulLoginActionOnMainPage(java.util.Map)[pri:0, instance:local.HLVTestApp.loginandsignuptests.PositiveLoginTests@14fc5f04]" started-at="2021-09-16T11:20:17 CEST" name="verifySuccessfulLoginActionOnMainPage" data-provider="csvReader" finished-at="2021-09-16T11:20:17 CEST" duration-ms="443" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[{no=2, description=valid username and password 2, password=testPassword2, username=testUsername2}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [http://localhost:8080/login] but found [http://localhost:8080/?username=testUsername2&password=testPassword2]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [http://localhost:8080/login] but found [http://localhost:8080/?username=testUsername2&password=testPassword2]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at local.HLVTestApp.loginandsignuptests.PositiveLoginTests.verifySuccessfulLoginActionOnMainPage(PositiveLoginTests.java:36)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifySuccessfulLoginActionOnMainPage -->
        <test-method is-config="true" signature="tearDownAfterEach()[pri:0, instance:local.HLVTestApp.loginandsignuptests.PositiveLoginTests@14fc5f04]" started-at="2021-09-16T11:20:17 CEST" name="tearDownAfterEach" finished-at="2021-09-16T11:20:18 CEST" duration-ms="664" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownAfterEach -->
      </class> <!-- local.HLVTestApp.loginandsignuptests.PositiveLoginTests -->
    </test> <!-- PositiveLoginOnMainPageTest2-FF -->
  </suite> <!-- PositiveLoginTestSuite -->
</testng-results>
